### ISSUE TITLE
how to extract TWO binary for region

### ISSUE THREAD

**Issue by ekg** [mklarqvist/Tomahawk#21]
I'm trying to extract .two format data in native binary format, however this always yields a text file:

```
tomahawk view -i in.calc_sorted.two -B -I tig00000855 -a 0 -f 2 >tig00000855.two
```
How do I output binary TWO format?

**Comment by mklarqvist** [comment]
This functionality was accidentally disabled in the last release. It will be made available again in the forthcoming release.

**Comment by mklarqvist** [comment]
This is now addressed in the new [beta-0.6.0](https://github.com/mklarqvist/tomahawk/releases/tag/beta-0.6.0) release
